### DISCUSSION TITLE
Single-axis move to edge

### DISCUSSION THREAD

**Discussion by bvandergiessen** [rxhanson/Rectangle#732] [Ideas]
It seems this is not yet an option: move current window to an edge along a single axis.

For example, moving it to the left edge of the screen without resizing and without moving it up or down (so, just along the X axis); moving it to the bottom by moving only along the Y axis, etc. These commands would allow better moving control without resizing, e.g. a window could be put in a corner in two steps.

The current _Move to Edge_ commands move the window along both axes, for instance _Move Up_ puts it at the top edge and also centers it horizontally.

Configuring the moving algorithm manually is not possible, is it?

**Comment by rxhanson** [comment]
The default behavior here is weird, I know. Here's how to change it to not center when moving:

https://github.com/rxhanson/Rectangle/blob/master/TerminalCommands.md#move-updownleftright-dont-center-on-edge

Restart the app after executing the terminal command.

**Reply by bvandergiessen** [reply]
Great, thanks. Needless to say, it might be useful to change what is default, and/or provide a GUI setting.
